test/ProblemDetailsResponseFactoryTest.php

```diff
@@ -13,13 +13,15 @@
 use RuntimeException;
 use Zend\ProblemDetails\Exception\InvalidResponseBodyException;
 use Zend\ProblemDetails\Exception\ProblemDetailsException;
-use Zend\ProblemDetails\ProblemDetailsResponse;
 use Zend\ProblemDetails\ProblemDetailsResponseFactory;
 
 class ProblemDetailsResponseFactoryTest extends TestCase
 {
     use ProblemDetailsAssertionsTrait;
 
+    private $request;
+    private $factory;
```

Comment:
**IF** we're going to declare this explicitly, then we should also include their types via a docblock.